libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -682,7 +677,8 @@ private void distanceText(InstructionModel model) {
    *
    * @param model provides instruction text
    */
-  private void updateTextInstruction(InstructionModel model) {
+  private void updateBannerData(InstructionModel model) {
+    updateManeuverView(model);
```

Comment:
Wondering if we should extract this call from here and make it explicit wherever we call `updateBannerData`. At a first glance `updateBannerData` and `updateManeuverView` seem completely different things to me. Could you clarify?

Reply:
The `ManeuverView` renders the arrow based on the `String` maneuver type + modifier data.  This is provided in the `InstructionModel` - maybe `updateManeuverViewData` would be more clear here?

Reply:
So the `ManeuverView` is part of the _Banner_, right? In terms of `View`s (overall layout), could you clarify what's the _Instruction_, _Banner_, _Maneuver_, etc.? I think it's confusing to me because I don't have the big picture of how it's all glued together.

Reply:
![device-2018-06-05-133454](https://user-images.githubusercontent.com/8434572/40993210-f5dd158a-68c6-11e8-9937-699914cf5a97.png)